app-tasks/rf/src/rf/uploads/geotiff/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .create_thumbnail import create_thumbnail
```

Comment:
This file can probably be deleted; it doesn't seem like there's much chance that we'll end up using it.